components/scheduler/src/index.html

```diff
@@ -22,15 +31,24 @@
         margin-left: 10vw;
         height: 80vh;
         margin-top: 10vh;
+
+        display: grid;
+        grid-template-columns: 2fr 1fr;
+        gap: 1rem;
       }
     </style>
   </head>
+
   <body>
     <main>
-      <nylas-scheduler
-        availability_id="demo-availability"
-        id="demo-scheduler"
-      ></nylas-scheduler>
+      <nylas-availability
+        show_as_week="true"
+        allow_booking="true"
+        max_bookable_slots="4"
+        id="demo-availability"
+      >
+      </nylas-availability>
+      <nylas-scheduler id="demo-scheduler"></nylas-scheduler>
```

Comment:
This rules! I wonder if it's possible for us to allow users to drop nylas-availability in as a slot within nylas-scheduler.... (a layout where we have something like an intro on the left, the availability picker in the middle, and detail confirmation on the right, like calendly etc.)